### ISSUE TITLE
Is this an official implementation?

### ISSUE THREAD

**Issue by twmht** [imenurok/ShakeDrop#1]
Is this an official implementation??

**Comment by imenurok** [comment]
Yes. This is an official implementation of ShakeDrop (https://arxiv.org/abs/1802.02375).

**Comment by twmht** [comment]
@imenurok 

I have read the paper,  but got some questions.

from the paper, you sample a location via Bernoulli distribution, and mask out 7x7 block centered at the location. However, since you use max pooling, there might exists a block which are not fully 7x7 zero. Am I right?

**Comment by imenurok** [comment]
While I defined the Max as max pooling, I did not use max pooling on the architecture.
